packages/issuer/contracts/Issuer.sol

```diff
@@ -7,22 +7,21 @@ import "@openzeppelin/contracts-ethereum-package/contracts/ownership/Ownable.sol
 import "./Registry.sol";
 
 contract Issuer is Initializable, Ownable {
-    event NewCertificationRequest(address indexed _owner, uint256 indexed _id);
+    event NewCertificationRequest(address indexed _owner, uint256 indexed _id, string indexed _deviceId);
```

Comment:
Ideally we would keep the naming scheme such as: 

"All events should be represented as verbs in the past tense such as CustomerRelocated, CargoShipped, etc."